### PR TITLE
live-common 19.10.0

### PR DESCRIPTION
**Pull request by gre**
**https://github.com/LedgerHQ/ledger-live-common/releases/tag/v19.10.0**

---

This includes incoming work of live-common + Ethereum + plugins system.

+ https://github.com/LedgerHQ/ledgerjs/pull/602
+ https://github.com/LedgerHQ/ledger-live-common/pull/1193

What should be evaluated is:

- non regression: that this PR will work with the existing Ethereum app (provider 1, production)
  - send ethereum.
  - send erc20.
  - compound flows.
  - that this PR also work with incoming Ethereum app (provider 4)
- this PR also includes latest fix from Juan regarding device flow
- device flows as non experimental install still works
  - specifically on swap
- device flows with experimental install works too
  - specifically on swap

### CODE REVIEW

**Comment by Arnaud97234** [comment]
**Eth 1.7.7**
- [ ] Send ethereum
- [ ] Send erc20
- [ ] Compound Deposit
- [ ] Compound Withdrawal
- [ ] Device flow
- [ ] Swap

**Comment by gre** [comment]
@Arnaud97234 to me the scope of this PR is only to validate the current stable app. not any future ETH app (it's bonus at least).

on my side i was able to validate these:

**LLD mac os & nano s 2.0 on app eth 1.7.7**
- send eth
- send erc20
- swap eth to usdt
- lending actions (lend, approve, withdraw, withdraw max) => it correctly show on device. **I did not broadcast them**

**Comment by Arnaud97234** [comment]
On my end I test Linux + NanoX 1.3.0 + Ethereum 1.7.7:
- Swap Eth to USDC -> Brodcasted
- Compound Withdraw -> Broadcasted
- Compound Deposit -> Not Broadcasted (went until validation step)
- Send erc20
- Send Eth
- Device flow seems ok as well

**Review by Arnaud97234 — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "@ledgerhq/hw-transport-http": "^5.51.1",
     "@ledgerhq/hw-transport-node-hid-singleton": "^5.51.1",
     "@ledgerhq/ledger-core": "6.11.3",
-    "@ledgerhq/live-common": "19.9.0",
+    "@ledgerhq/live-common": "19",
```
Comment (juan-cortes) [inline package.json:49]: 19.10.0? @gre

**Review by juan-cortes — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (gre) [reply package.json:49]: it's not a problem. only yarn.lock is the source of trust for versions. "19" is equivalent to `^19.0.0` which is legit match. but package.json is not really the truth.

**Review by nabil-brn — APPROVED** [review]
MacOS / NanoS / ETH app 1.7.8

- [x] ETH, ERC20, ETC, DOT, XRP, XLM, BTC, BCH 
- [x] Compound flows (not broadcasted)
- [x] Swap Doge -> ETH
- [x] Inline app install

---
7183e67c3 (HEAD -> live-common-20-alpha, gre/live-common-20-alpha) snapshot

**Review by MortalKastor — APPROVED** [review]

**Pull request merged**